src/pep8ext_naming.py

```diff
@@ -448,6 +453,8 @@ class VariablesCheck(BaseASTCheck):
     def _find_errors(self, assignment_target, parents, ignore):
         for parent_func in reversed(parents):
             if isinstance(parent_func, ast.ClassDef):
+                if "TypedDict" in parent_func.superclasses:
```

Comment:
Why are we skipping this altogether? People may in fact want to check this. I would instead think we should add an error code for this case so it can be ignored/disabled on a case-by-case basis

Reply:
Good question. Happy to have a go at implementing that. Would you go with something like `N819: mixedCase variable in TypedDict`? Do you think we should disable by default?

Reply:
Have pushed a new commit to implement this new error code, let me know what you think